engli/106_modologe.md

```diff
@@ -1,40 +1,46 @@
 # Adverbs
 
-## Modifying a verb
+There are two types of adverbs in Pandunia,
+adverbs of degree
+and adverbs of comparison.
 
-An adverb is a word that modifies a verb or another modifier.
+## Adverbs of degree
 
-In Pandunia adverbs end in **-o** and their place is before a verb
-or at the end of the sentence.
+Adverbs of degree tell us about the intensity of something.
 
-**me _bono_ sona.**
-– I well sleep.  
-**me sona _bono_.**
-– I sleep _well_.  
-**te baxa pandunia _bono_.**
-– You speak Pandunia _well_.
+**tre**
+– very  
+**tai**
+– too  
+**bas**
+– quite, sufficiently, enough  
+**max**
+– more  
+**min**
+– less
 
-## Modifying another modifier
+Adverbs of degree are placed before the adjective that they modify.
 
-A modifier can modify also another modifier.
-For example **day** (big) and **lil** (little) can be used as intensifying adverbs.
+**un dai man**
+– a big man  
+**un tre dai man**
+– a very big man  
+**un tai dai man**
+– a too big man  
+**un bas dai man**
+– a big enough man / a sufficiently big man
 
-**dayi jan**
-– a big person  
-**lili jan**
-– a little person  
-**sundari jan**
-– a beautiful person
+The adverb and adjective can together modify a verb or a phrase.
 
-**dayo sundari jan**
-– very beautiful person  
-**lilo sundari jan**
-– somewhat beautiful person
+**mo _tre bon_ sona.**
+– I very well sleep.  
+**mo sona _tre bon_.**
+– I sleep _very well_.  
```

Comment:
But cannot one say: "mo sona bon" – 'I sleep well'? I don't think all adverbs will include a degree word?

Reply:
In practice, yes, one could say it like that too. I prefer to use a degree adverb together with adjectives because degree adverbs are structure words that help to identify the roles of content words. A series of content words can be interpreted in many different ways, as I tried to demonstrate in our debate in Reddit, though I must admit that this specific sentence is not likely to be misunderstood.